Chapter5/stats_handler.erl

```diff
@@ -0,0 +1,13 @@
+-module(stats_handler).
```

Comment:
The user is responsible for init Db as I followed the same thing the example did with the log_handler and the io_handler, so its initialized and added the same way they are: event_manager:add_handler(EventHandler, {stats_handler, []}).

If you think I should change how the stats_handler is initialized or added let me know and I can update the code